system/languages/es.json

```diff
@@ -0,0 +1,203 @@
+{
+    "FateX": "FateX",
+    "FAx": {
+        "name": "FateX",
+        "Global": {
+            "Ladder": {
+                "+8": "Legendario",
+                "+7": "Epico",
+                "+6": "Fantastico",
+                "+5": "Excelente",
+                "+4": "Enorme",
+                "+3": "Grande",
+                "+2": "Bueno",
+                "+1": "Normal",
+                "+0": "Mediocre",
+                "-1": "Malo",
+                "-2": "Terrible",
+                "-3": "Catastrófico",
+                "-4": "Horrible"
+            },
+            "Actions": {
+                "Configure": "Configurar",
+                "Delete": "Eliminar",
+                "Duplicate": "Duplicar",
+                "Sort": "Ordenar",
+                "Create": "Crear actor",
+                "Choose": "Escoger plantilla"
+            },
+            "BaseItem": "Item base"
+        },
+        "Sheet": {
+            "Name": "Nombre",
+            "Description": "Descripción",
+            "FatePoints": "Puntos Fate",
+            "Aspects": "Aspectos",
+            "Consequences": "Consecuencias",
+            "Skills": "Habilidades",
+            "Stunts": "Proezas",
+            "Extras": "Extras",
+            "Bio": "Bio",
+            "Artwork": "Arte",
+            "StressTracks": "Medidores de estrés",
+            "Tabs": {
+                "SkillsStunts": "Habilidades y Proezas",
```

Comment:
I'd remove the capital P from Proezas (in Spanish these nouns are not usually capitalized, even if they refer to other fields).